zipkin-lens/src/test/data/malformed.json

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    "traceId": "6cf5624e3452d1bc202a57dad35921c0",
+    "parentId": "d65495313f944801",
+    "id": "d5cb70daead478dc",
+    "name": "kong.header_filter",
+    "timestamp": 1570005203864000
+  },
+  {
+    "traceId": "6cf5624e3452d1bc202a57dad35921c0",
+    "parentId": "d65495313f944801",
+    "id": "80b8efd52d66af27",
+    "name": "kong.body_filter",
+    "timestamp": 1570005203864000
```

Comment:
this should have been an annotation

Reply:
These are spans in Kong; I recall that when they execute in under a millisecond then they appear as a 0-length span in zipkin.

Reply:
minimum duration is 1us, so prefer to use that vs rounding down to zero, especially if precision is millis (1000us) on the kong side. Zero implies incomplete.

The main thing is to differentiate annotations (timestamped events on a timeline) from spans (some wrapped thing with a duration).